### ISSUE TITLE
Unexpected result for /service-worker.js

### ISSUE THREAD

**Issue by Cadynum** [photoview/photoview#1023]
**Describe the bug**
During loading photoview, firefox shows a warning in the console:

`Error during service worker registration: DOMException: The operation is insecure.`

Chrome shows:
```
The script has an unsupported MIME type ('text/html').
index.8d4c57fa.js:224 Error during service worker registration: DOMException: Failed to register a ServiceWorker for scope ('https://...') with script ('https://.../service-worker.js'): The script has an unsupported MIME type ('text/html').
```

When trying to inspect the file, it looks like it doesn't exist at all, and the entry-page is returned:
 
```
$ curl -i 127.0.0.1:3441/service-worker.js


HTTP/1.1 200 OK
Accept-Ranges: bytes
Content-Length: 864
Content-Type: text/html; charset=utf-8
Last-Modified: Thu, 08 Aug 2024 19:31:03 GMT
Vary: Accept-Encoding
Date: Sun, 18 Aug 2024 15:10:02 GMT

<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <link rel="icon" href="/photoview-logo.svg" type="image/svg+xml" />
    <link rel="apple-touch-icon" href="/logo192.png" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="theme-color" content="#000000" />

    <link rel="manifest" href="/manifest.json" />

    <meta name="apple-mobile-web-app-title" content="Photoview" />
    <meta name="apple-mobile-web-app-capable" content="yes" />
    <meta name="apple-mobile-web-app-status-bar-style" content="white" />

    <script type="module" crossorigin src="/assets/index.1d51bb18.js"></script>
    <link rel="stylesheet" href="/assets/index.0ac106ed.css">
  </head>
  <body>
    <noscript>You need to enable JavaScript to run Photoview.</noscript>
    <div id="root"></div>
    
  </body>
</html>
```

**Your environment:**
  * Running on Raspberry Pi OS for Raspberry Pi 5
  * Using this version on Docker arm64: https://github.com/photoview/photoview/pull/1014

**Comment by kkovaletp** [comment]
@Cadynum, thanks for the bug report and your investigation.
Actually, we already have this issue reported with some investigation done and even several tries to fix it in different ways: #938 
Unfortunately, we currently don't have a UI developer, who wants to take this issue and fix it, so we have to wait...

Closing this issue as a duplicate. Feel free to add your thoughts to the mentioned one.

**Comment by Cadynum** [comment]
@kkovaletp I understand. Thanks for clarifying.
